types/zxcvbn/tsconfig.json

```diff
@@ -14,7 +14,8 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
Without this option, type inference will not work.

<img width="618" alt="Screen Shot 2021-03-13 at 10 37 09 PM" src="https://user-images.githubusercontent.com/5278201/111031826-c73d5b00-844c-11eb-80fe-afe1a4313ac5.png">